FirebaseFirestore.podspec

```diff
@@ -16,7 +16,7 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
                        DESC
 
   s.homepage         = 'https://developers.google.com/'
-  s.license          = { :type => 'Apache', :file => '../LICENSE' }
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
   s.authors          = 'Google, Inc.'
 
   s.source           = { :git => 'https://github.com/TBD/Firestore.git', :tag => s.version.to_s }
```

Comment:
There is no `0.9.2` tag; this doesn't seem like it could pass `pod lib lint`.

Reply:
`pod lib lint` is only a local check. The missing tag is an issue for `pod spec lint` though.  

Separately, we won't be able to run `pod spec lint` until we can push the dependencies. The public FirebaseCore is not satisfactory, since the open source components depend on the open source Core for private header resolution.

`pod lib lint` only works for FirebaseCore because `pod lib lint` also requires dependencies to be publicly pushed. There are some possible approaches to address this, but depending on priorities, it might make sense to punt on linting until we're ready to do official releases from open source and in the meantime monitor warnings in the Xcode builds.